### ISSUE TITLE
A workaround that bypasses challenge's limitation of not using `endsWith()` function.

### ISSUE THREAD

**Issue by tanmay7270** [FreeCodeCamp/FreeCodeCamp#12198]
#### Challenge Name

<a href = "https://www.freecodecamp.com/challenges/confirm-the-ending">Confirm the Ending</a>
#### Issue Description

**TL;DR:** Start reading after **Browser Information**.

**Detailed Description:**
FreeCodeCamp requires the camper to not use `.endsWith()` function to solve the given _Basic Algorithm Scripting_ challenge in any way:
> This challenge can be solved with the .endsWith() method, which was introduced in ES2015. But for the purpose of this challenge, we would like you to use one of the JavaScript substring methods instead.

However, a simple workaround can bypass this given test: 

> Do not use the built-in method `.endsWith()` to solve the challenge.

Workaround: Using `if` and `else` conditions to return `true` or `false` if the condition of `str.endsWith(target)` is satisfied.

Important Note: I noticed that this happened only until I used this workaround first, and then as soon as I used the `.endsWith()` function method without the workaround once (which is bound to fail the test), the workaround stopped passing the test as well.

#### Browser Information

* Browser Name, Version: Google Chrome Canary, 57.0.2960.0 
* Operating System: OS X El Capitan (Version: 10.11.6)
* Mobile, Desktop, or Tablet: Macbook Pro 13' Retina (Early 2015)

#### Your Code

```js
 function confirmEnding(str, target) {

  // #BUG: A workaround to bypass the limitation of not being able to use built-in function `.endsWith()`
  
  if (str.endsWith(target)) {
    return true;
  } else {
    return false;
  }
}

confirmEnding("Bastian", "n");

 
 
```
^This code passes the challenge and uses the built-in method `.endsWith()`. If the code doesn't work, please click on _Reset_ button and paste it again. 
#### Screenshot

<img width="1280" alt="screen shot 2016-12-24 at 1 14 47 pm" src="https://cloud.githubusercontent.com/assets/11278685/21466078/8ffa65f4-c9e1-11e6-81c3-fe9c46b85d5d.png">

**Comment by erictleung** [comment]
@tanmay7270 thanks for the issue. This issue should be fixed in the `staging` branch of FCC from this PR https://github.com/FreeCodeCamp/FreeCodeCamp/pull/10286. This just hasn't been pushed up to the production  site yet. Closing as resolved in staging. Happy coding!
